corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/api.js

```diff
@@ -67,6 +71,15 @@ hqDefine("cloudcare/js/formplayer/menus/api", [
                         return;
                     }
                 }
+                let appId = params.appId;
+                if (!params.preview) {
+                    if (app.get('copy_of') && app.get('copy_of') === params.appId) {
+                        appId = app.id;
+                    }
+                    if (app.get('copy_of') && app.get('copy_of') === params.copyOf) {
+                        appId = params.appId;
```

Comment:
I guess this is redundant with the default `let appId = params.appId;` so can be deleted